VoodooPS2Trackpad/VoodooPS2SynapticsTouchPad.h

```diff
@@ -268,6 +269,10 @@ class EXPORT ApplePS2SynapticsTouchPad : public IOHIPointing
 
 	ForceTouchMode _forceTouchMode {FORCE_TOUCH_BUTTON};
 	int _forceTouchPressureThreshold {100};
+
+    int _forceTouchCustomDownThreshold;
+    int _forceTouchCustomUpThreshold;
+    int _forceTouchCustomPower;
```

Comment:
Please initialize it to some default usable values. I can't test it because I don't know what values should I use.

Reply:
#14